temba/locations/migrations/0003_adminboundary_in_country.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('locations', '0002_auto_20141126_2054'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='adminboundary',
+            name='in_country',
+            field=models.CharField(
+                help_text=b"The OSM id of this admin level's country id", max_length=15, null=True),
+            preserve_default=True,
```

Comment:
@nicpottier it is not an MPTT thing, i added it so that i can easily tell if an admin level is within a country without running nested query from the level up to the country level. 

e.g https://github.com/rapidpro/rapidpro/blob/master/temba/locations/management/commands/import_geojson.py#L102

` if level == STATE_LEVEL:
            AdminBoundary.objects.filter(level=level, parent__osm_id=country_osm_id).exclude(osm_id__in=seen_osm_ids).delete()
        elif level == DISTRICT_LEVEL:
            AdminBoundary.objects.filter(level=level, parent__parent__osm_id=country_osm_id).exclude(osm_id__in=seen_osm_ids).delete()
`